CMakeLists.txt

```diff
@@ -755,33 +740,32 @@ if(AVIF_BUILD_APPS)
             LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
         )
     endif()
-    if(AVIF_ENABLE_GAIN_MAP)
-        include(LocalLibargparse)
-        set(AVIFGAINMAPUTIL_SRCS
-            apps/avifgainmaputil/avifgainmaputil.cc
-            apps/avifgainmaputil/convert_command.cc
-            apps/avifgainmaputil/combine_command.cc
-            apps/avifgainmaputil/extractgainmap_command.cc
-            apps/avifgainmaputil/imageio.cc
-            apps/avifgainmaputil/printmetadata_command.cc
-            apps/avifgainmaputil/tonemap_command.cc
-            apps/avifgainmaputil/program_command.cc
-            apps/avifgainmaputil/swapbase_command.cc
-        )
+    include(LocalLibargparse)
+
+    set(AVIFGAINMAPUTIL_SRCS
+        apps/avifgainmaputil/avifgainmaputil.cc
+        apps/avifgainmaputil/convert_command.cc
+        apps/avifgainmaputil/combine_command.cc
+        apps/avifgainmaputil/extractgainmap_command.cc
+        apps/avifgainmaputil/imageio.cc
+        apps/avifgainmaputil/printmetadata_command.cc
+        apps/avifgainmaputil/tonemap_command.cc
+        apps/avifgainmaputil/program_command.cc
+        apps/avifgainmaputil/swapbase_command.cc
+    )
 
-        add_executable(avifgainmaputil "${AVIFGAINMAPUTIL_SRCS}")
-        if(WIN32)
-            if(MSVC)
-                target_sources(avifgainmaputil PRIVATE apps/utf8.manifest)
-            elseif(MINGW)
-                target_sources(avifgainmaputil PRIVATE apps/utf8.rc)
-            endif()
+    add_executable(avifgainmaputil "${AVIFGAINMAPUTIL_SRCS}")
+    if(WIN32)
+        if(MSVC)
+            target_sources(avifgainmaputil PRIVATE apps/utf8.manifest)
+        elseif(MINGW)
+            target_sources(avifgainmaputil PRIVATE apps/utf8.rc)
         endif()
-        set_target_properties(avifgainmaputil PROPERTIES LINKER_LANGUAGE "CXX")
-        target_include_directories(avifgainmaputil PRIVATE apps/avifgainmaputil/)
-        target_link_libraries(avifgainmaputil libargparse avif_apps avif avif_enable_warnings)
-        # Don't add avifgainmaputil to installed apps for now.
     endif()
+    set_target_properties(avifgainmaputil PROPERTIES LINKER_LANGUAGE "CXX")
+    target_include_directories(avifgainmaputil PRIVATE apps/avifgainmaputil/)
+    target_link_libraries(avifgainmaputil libargparse avif_apps avif avif_enable_warnings)
+    # Don't add avifgainmaputil to installed apps for now.
```

Comment:
I leave this decision to you. Just wanted to confirm this is still the case.

If `avifgainmaputil` calls only public API functions of libavif, it seems fine to add it to installed apps.

Reply:
Makes sense. I will leave this for another PR though.